README.md

```diff
@@ -25,7 +25,7 @@ Use this SDK to build Watson-powered applications in Unity.
 ## Before you begin
 Ensure that you have the following prerequisites:
 
-* An IBM Cloud account. If you don't have one, [sign up][ibm_cloud_registration].
+* You need an [IBM Cloud][ibm-cloud-onboarding] account.
 * [Unity][get_unity]. You can use the **free** Personal edition.
 * Change the build settings in Unity (**File > Build Settings**) to any platform except for web player/Web GL. The Watson Developer Cloud Unity SDK does not support Unity Web Player.
```

Comment:
Not part of this change set, and probably not a big deal, but it doesn't make sense here because it's a follow on from the bullet above. I would suggest adding a specific statement about configuring unity later on.